vms/registration/views.py

```diff
@@ -2,106 +2,111 @@
 from django.core.urlresolvers import reverse
 from django.http import HttpResponseRedirect
 from django.shortcuts import render
-
+from braces.views import LoginRequiredMixin, AnonymousRequiredMixin
+from django.views.generic.edit import FormView
+from django.views.generic import TemplateView
+from django.core.urlresolvers import reverse_lazy
 from administrator.forms import AdministratorForm
 from organization.services import (get_organizations_ordered_by_name,
                                    get_organization_by_id)
 from volunteer.forms import VolunteerForm
 from volunteer.validation import validate_file
 from registration.forms import UserForm
 from registration.phone_validate import validate_phone
+from administrator.models import *
 
 
-def signup_administrator(request):
+class AdministratorSignupView(TemplateView):
     """
-    This method is responsible for diplaying the register user view
-    Register Admin or volunteer is judged on the basis of users
-    access rights.
-    Only if user is registered and logged in and registered as an
-    admin user, he/she is allowed to register others as an admin user
```

Comment:
@amruthasangeeth Can you please add the removed part of this comment back? Is there any specific reason you have removed it?